### PR TITLE
Feature/UI devices

### PR DESCRIPTION
**Pull request by Malak67**
Created device list page

### CODE REVIEW

**Review by alexworker23 — COMMENTED** [review]
1) In `.env.example` you have backend port 3040, but for UI you have backend url http://localhost:3030/. 
2) When unlogged - you are sending request to `api/invitation`. 
This should be disabled by passing `enabled` to fetching hook.

File: apps/drec-ui/src/device/containers/PublicDeviceCard/PublicDeviceCard.styles.ts [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    card: {
+        height: '100%',
+        minWidth: 240
+    },
+    icon: {
+        width: 80,
+        padding: 50,
+        fill: theme.palette.primary.main,
+        '&:hover': {
+            color: 'red !important'
```
[the remark this answers is not in the document]
Comment (Malak67) [reply apps/drec-ui/src/device/containers/PublicDeviceCard/PublicDeviceCard.styles.ts:13]: Forgot about this.. It was just for testing

File: apps/drec-ui/src/device/containers/PublicDeviceCard/PublicDeviceCard.styles.ts [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+        }
+    },
+    imageWrapper: {
+        color: '#fff !important'
```
[the remark this answers is not in the document]
Comment (Malak67) [reply apps/drec-ui/src/device/containers/PublicDeviceCard/PublicDeviceCard.styles.ts:17]: Yep

File: apps/drec-ui/src/device/containers/PublicDeviceCard/PublicDeviceCard.tsx [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+    });
+
+    const classes = useStyles();
+
+    return (
+        <CardWithImage
+            {...cardProps}
+            fallbackIconProps={{ className: classes.icon }}
+            cardProps={{ className: classes.card }}
+            overlayTextProps={{ className: classes.imageWrapper }}
```
[the remark this answers is not in the document]
Comment (Malak67) [reply apps/drec-ui/src/device/containers/PublicDeviceCard/PublicDeviceCard.tsx:26]: Agree

**Review by alexworker23 — APPROVED** [review]

**Pull request merged**